code/__HELPERS/mobs.dm

```diff
@@ -207,10 +207,6 @@ proc/add_ghostlogs(var/mob/user, var/obj/target, var/what_done, var/admin=1, var
 			L = list("chest", "groin", "l_arm", "r_arm", "l_leg", "r_leg")
 		if(slot_gloves, slot_handcuffed) //Gloves
 			L = list("l_hand", "r_hand")
```

Comment:
Shouldn't be, until somebody reworks cuffs and gloves (which won't happen until somebody reworks inventory) they are going to be somewhat hardcoded